test/configurationTests.ts

```diff
@@ -343,68 +62,144 @@ describe("Configuration", () => {
     });
 
     describe("loadConfigurationFromPath", () => {
+        const externalCustomRulesConfig = buildConfig({
+            jsRules: new Map<string, Partial<IOptions>>([
+                ["rule-one", {}],
+                ["rule-two", { ruleSeverity: "off" as RuleSeverity }],
+            ]),
+            rules: new Map<string, Partial<IOptions>>([
+                ["rule-one", {}],
+                ["rule-two", { ruleSeverity: "off" as RuleSeverity }],
+            ]),
+        });
+
+        it("overrides defaultSeverity of base configs", () => {
```

Comment:
These three tests became much shorter because now all that matters is that the base config gets the correct severity.